src/AccessibilityInsights.SetupLibrary/VersionSwitcherWrapper.cs

```diff
@@ -99,8 +99,9 @@ private static bool TryRecursiveCopy(string source, string dest, List<FileStream
                 foreach (string file in Directory.GetFiles(source))
                 {
                     FileInfo fileInfo = new FileInfo(file);
-                    fileInfo.CopyTo(Path.Combine(dest, file), true);
-                    fileLocks.Add(File.OpenRead(dest));
+                    string destFile = Path.Combine(dest, fileInfo.Name);
+                    fileInfo.CopyTo(destFile, true);
+                    fileLocks.Add(File.OpenRead(destFile));
```

Comment:
>destFile [](start = 48, length = 8)

Old code was trying to open `dest`, which is the folder, not the file. Changed this to use `destFile`, which is the file itself.